src/Dialect/ONNX/ONNX.td

```diff
@@ -244,7 +244,7 @@ def ONNX_SeqType : ONNX_Type<"Seq", "Seq"> {
     $_printer << "<" << getImpl()->ElementType << ">";
   }];
   let parser = [{
```

Comment:
Looks like I missed one! We should convert this to not user parser anymore either.

Reply:
OK we can do that in another PR.